src/code42cli/output_formats.py

```diff
@@ -10,16 +10,29 @@
 CEF_DEFAULT_SEVERITY_LEVEL = "5"
 
 
-class OutputFormat:
-    TABLE = "TABLE"
-    CSV = "CSV"
+class JsonOutputFormat:
     JSON = "JSON"
     RAW = "RAW-JSON"
 
+    def __iter__(self):
+        return iter([self.JSON, self.RAW])
+
+
+class OutputFormat(JsonOutputFormat):
+    TABLE = "TABLE"
+    CSV = "CSV"
+
     def __iter__(self):
         return iter([self.TABLE, self.CSV, self.JSON, self.RAW])
 
 
+class SendToOutputFormat(JsonOutputFormat):
```

Comment:
Perhaps this should be called `SendToFileEventsOutputFormat` since it doesn't apply to alerts